src/components/ebay-filter-menu-button/index.js

```diff
@@ -81,19 +67,19 @@ module.exports = require('marko-widgets').defineComponent({
         this.buttonEl.setAttribute('aria-expanded', false);
         this.buttonEl.setAttribute('aria-pressed', (this.getCheckedItems().length > 0));
     },
-    handleFormSubmit(originalEvent) {
-        this.emitComponentEvent('form-submit', null, originalEvent);
+    handleFormSubmit(e) {
+        this.emitComponentEvent('form-submit', null, e.originalEvent);
+        e.originalEvent.preventDefault();
```

Comment:
I don't think we should prevent default here. That should be left to the end user since they have access to the original event. As it stands there is no way to have the built in form behavior with this component.

Reply:
This was left in by mistake! Fixing.